src/neoxp/Commands/TransferNFTCommand.cs

```diff
@@ -76,19 +76,15 @@ internal async Task<int> OnExecuteAsync(CommandLineApplication app, IConsole con
                 return 1;
             }
 
-            static string HexStringToUTF8(string hex)
+            static string HexOrBase64ToUTF8(string input)
             {
-                hex = hex.ToLower().Trim();
-                if (!new Regex("^(0x)?([0-9a-f]{2})+$").IsMatch(hex))
-                    throw new FormatException();
-
-                if (new Regex("^([0-9a-f]{2})+$").IsMatch(hex))
+                try
                 {
-                    return Encoding.UTF8.GetString(hex.HexToBytes());
+                    return input.StartsWith("0x") ? Encoding.UTF8.GetString(input[2..].HexToBytes().Reverse().ToArray()) : Encoding.UTF8.GetString(Convert.FromBase64String(input));
```

Comment:
Are we sure `reverse` is appropriate?

Reply:
I'm sure, it starting with 0x is big-endian hex string, need to remove 0x and revert to byte array. Same deal [here](https://github.com/neo-project/neo-express/blob/5e72e0c70944579fd82dd37d6a601a3de0f70229/src/neoxp/Commands/ShowCommand.NFT.cs#L68C25-L68C34).

Reply:
but that is only for `UInt160`. Normally with just `hexstring` it isnt reversed. I know people complained about this. But we can leave for now.

Reply:
Maybe a note in the documentation would be nice like [this](https://github.com/neo-project/neo-dev-portal/blob/dd4616bd3d088ce8fa365c55d37f7f70b52cd77a/docs/n3/exchange/transaction.md?plain=1#L380)

```
NOTE
In Neo, hexadecimal strings are processed in big-endian order if they are preceded by 0x, or little-endian order if they are not.
```

Reply:
Yes, `show nft` is reverse too, https://github.com/neo-project/neo-express/blob/5e72e0c70944579fd82dd37d6a601a3de0f70229/src/neoxp/Commands/ShowCommand.NFT.cs#L68C25-L68C34